### PR TITLE
Doc/jsdoc

### PR DESCRIPTION
**Pull request by matiu**
adds `npm run-script docs`

to update README.md with JSDOC documentation.

Also updates and extends `irene` & `tomas` examples.

Related: https://github.com/bitpay/bitcore-wallet-client/issues/227#issuecomment-202470557

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,311 +121,994 @@ info Generating new keys
...
+
+
+
+* * *
+
+## Class: API
 ClientAPI constructor.
 
-**Params**
+### API.API.API#setNotificationsInterval(notificationIntervalSeconds)
```
Comment (isocolsky) [inline README.md]: `API.API.API`....

**Comment by matiu** [comment]
updated.

**Comment by matiu** [comment]
updated (fix name space in source files)

**Comment by isocolsky** [comment]
A much needed PR :) Great job!

**Pull request merged**